python/pyraphtory/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pyraphtory"
-version = "0.2.0a1"
+version = "0.2.0a5"
```

Comment:
Can these versions be standardised in next PR?

Reply:
yeah sure